res/css/structures/_RightPanel.scss

```diff
@@ -177,14 +182,14 @@ $pulse-color: $alert;
 
 .mx_RightPanel_headerButton_unread {
     &::before {
-        background-color: $room-icon-unread-color !important;
+        background-color: $room-icon-unread-color  !important;
```

Comment:
why double-space?

Reply:
I had the VSCode autoformatter for SCSS enabled. That messed up a couple of stuff. reverting all of that